linux/docker_agent.jl

```diff
@@ -0,0 +1,48 @@
+using RootfsUtils: parse_build_args, debootstrap, chroot, upload_gha, test_sandbox
+
+args         = parse_build_args(ARGS, @__FILE__)
+arch         = args.arch
+archive      = args.archive
+image        = args.image
+
+@info arch
+# Build debian-based image with the following extra packages:
+packages = [
+    "bash",
+    "curl",
+    "locales",
+    "vim",
+    "keychain",
+    "git",
+    "procps",
+    "build-essential",    
+    "apt-transport-https" ,
+    "curl" ,
+    "gnupg-agent",
+    "gnupg",
+    "software-properties-common",
+    "unzip",
+]
+
+artifact_hash, tarball_path = debootstrap(arch, image; archive, packages) do rootfs, chroot_ENV
+    my_chroot(args...) = chroot(args...; ENV=chroot_ENV, uid=0, gid=0)
+
+    @info("Installing AWS cli v2...")
+    awscliv2_install_cmd = """
+    cd /usr/bin
+    curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
+    unzip awscliv2.zip
+    ./aws/install
+    """
+    my_chroot(rootfs, "bash", "-c", awscliv2_install_cmd)
```

Comment:
Can you not just use `awscli` from `apt`?

Reply:
Using just the `awscli` resulted in
```
aws --version
aws-cli/1.16.113 Python/3.7.3 Linux/5.4.0-89-generic botocore/1.12.103
```
(while using the rootfs-image)

Reply:
Ah, so you need `aws` version 2+?

Reply:
Yes. Few of the AWS ECR related things like login from a docker etc run smoother with v2+